instrumental/drivers/motion/apt.py

```diff
@@ -11,8 +11,9 @@
 from ... import Q_, u
 
 
-def list_instruments():
-    return [ParamSet(TDC001_APT, port=p.device) for p in comports() if (p.vid, p.pid) == (0x0403, 0xfaf0)]
+def list_instruments(filter_serial_number=None):
+    return [ParamSet(TDC001_APT, port=p.device, serialnumber=p.serial_number) for p in comports() 
```

Comment:
Can you change `serialnumber` to `serial`? That will make it consistent with other devices.